### PR TITLE
Initial commit of Prometheus config and Grafana dashboards

### PR DESCRIPTION
**Pull request by james-laing**
This is the initial commit of example Prometheus.yaml, Grafana FlashArray Overview dashboard and readme to allow customers to monitor their fleet on-premise.

### CODE REVIEW

**Review by sdodsley — COMMENTED** [review]
Please put a link into the main README linking to this README so it is easy to find.

File: README.md [hunk 1]
```diff
@@ -109,3 +109,17 @@ docker run -d -p 3000:3000 --name=grafana -v /tmp/grafana-data:/var/lib/grafana
 Please have a look at the documentation of each image/application for adequate configuration examples.
 
 A simple but complete example to deploy a full monitoring stack on kubernetes can be found in the [examples](examples/config/k8s) directory
+
+## Monitoring On-Premise with Prometheus and Grafana
+Take holistic overview of your Pure Storage FlashArray estate on-preimise with Prometheus and Grafana to summerise statistics such as:
```
Comment (sdodsley) [inline README.md:114]: on-premise

Comment (sdodsley) [inline README.md:114]: Take a holistic...

File: extra/grafana/README.md [hunk 2]
```diff
@@ -14,7 +12,21 @@ This exporter is provided under Best Efforts support by the Pure Portfolio Solut
...
+  * Data Reduction Rate
+  * Number and type of open alerts
+
+Drill down into specific arrays and identify top busy hosts while correlating read and write operations and throughput to quickly highlight or elimiate investigation equiries.
+<br>
+<img src="./images/grafana_purefa_overview_dash_1.png" width="66%" height="66%">
+<img src="./images/grafana_purefa_overview_dash_2.png" width="33%" height="33%">
+<br>
+
+These dashboards are provide an overview of your fleet to give you early indications of potential issues and a look back in time of recent history. Once you are pulling the metrics, you can create your own dashboards bespooke to your environment, even correlating metrics from other technologies.
```
Comment (sdodsley) [inline extra/grafana/README.md:27]: remove 'are'.
...in time to recent...
bespoke, not bespooke

File: README.md (hunk 1 above)
Comment (james-laing) [reply README.md]: Thanks for catching these @sdodsley - corrections made in commit#3

Comment (james-laing) [reply README.md]: Thanks for catching these @sdodsley - corrections made in commit#3

File: extra/grafana/README.md [hunk 3]
```diff
@@ -14,7 +12,21 @@ This exporter is provided under Best Efforts support by the Pure Portfolio Solut
 5. Check out the features and default values set in the [Pure Storage FlashArray Overview Grafana Dashboard](#Pure-Storage-FlashArray-Overview-Grafana-Dashboard)
 
 # Overview
-So you would like to collect metrics from your Pure Storage FlashArray and monitor your environment on-premise? Well let's walk through it.
+Take holistic overview of your Pure Storage FlashArray estate on-preimise with Prometheus and Grafana to summerise statistics such as:
```
[the remark this answers is not in the document]
Comment (james-laing) [reply extra/grafana/README.md]: Thanks for catching these @sdodsley - corrections made in commit#3

File: extra/grafana/README.md (hunk 2 above)
Comment (james-laing) [reply extra/grafana/README.md]: Thanks for catching these @sdodsley - corrections made in commit#3

File: extra/grafana/README.md [hunk 4]
```diff
@@ -14,7 +12,21 @@ This exporter is provided under Best Efforts support by the Pure Portfolio Solut
...
+
+Drill down into specific arrays and identify top busy hosts while correlating read and write operations and throughput to quickly highlight or elimiate investigation equiries.
+<br>
+<img src="./images/grafana_purefa_overview_dash_1.png" width="66%" height="66%">
+<img src="./images/grafana_purefa_overview_dash_2.png" width="33%" height="33%">
+<br>
+
+These dashboards are provide an overview of your fleet to give you early indications of potential issues and a look back in time of recent history. Once you are pulling the metrics, you can create your own dashboards bespooke to your environment, even correlating metrics from other technologies. 
+
+If you would like to collect metrics from your Pure Storage FlashArray fleet and monitor your environment on-premise? Let's walk through it.
```
[the remark this answers is not in the document]
Comment (james-laing) [reply extra/grafana/README.md]: Thanks for catching these @sdodsley - corrections made in commit#3

**Review by sdodsley — COMMENTED** [review]
your image pure_om_exporters_prometheus_grafana.png has the FA and FB items switched. You have the FB OME pointing to an FA and vice-versa

**Review by sdodsley — APPROVED** [review]

**Review by sdodsley — COMMENTED** [review]
In the grafana_purefa_simple_query.png image, at the bottom, in the Job column, the array name still needs to be greyed out

**Comment by james-laing** [comment]
Issue resolved with last commit.

**Review by sdodsley — APPROVED** [review]

**Pull request merged**